### DISCUSSION TITLE
What functions need to be enabled?

### DISCUSSION THREAD

**Discussion by NARUTOfzr** [NARUTOfzr/Neptune_3#18] [Ideas]
What functions need to be enabled?

**Comment by xav-ie** [comment]
I think this commit is super helpful: https://github.com/Piets/Neptune_3/commit/42a78757cc19da64b5c44fc6d0170b694ad99b3e

**Comment by egs2003** [comment]
linear advance shoud be enabled. Also the possibility to change steps xyze on screen. I also like to see the current mesh without leveling. See and Change Z-Offset in Menu. The possibility to format the sd card. Do a system test of all fans. Single point test of Mesh. That would be helpful If i wanne adjust the bed by changing physics (x-axis, bed screws, etc).

**Reply by mlee12382** [reply]
Specifically Linear Advance with K = 0 so it doesn't affect anyone that doesn't want to use it. 
I know the screen firmware can support steps and pid values etc as the mks version of the screen has that functionality. 
Not sure on viewing the mesh outside of leveling or single point testing from the screen but enabling them to work with pronterface or octoprint or something should be doable at least.
